README.md

```diff
@@ -54,6 +54,7 @@ Bookstore CMS is a web application that allows users to manage their bookstore.
 - [VSCode](https://code.visualstudio.com/)
 - [react router](https://reactrouter.com/)
 - [react redux](https://react-redux.js.org/)
+- [react icons](https://react-icons.github.io/react-icons/)
```

Comment:
- [ ] I think it would be great if you add more future features in your README since you have one but you should have 1 to 3 so please add it and add more than one point  in key features section 🔥💯🤩